arcgis-ios-sdk-samples/Content Display Logic/FileTypes.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>Geodatabases</key>
+	<array>
+		<string>geodatabase</string>
+	</array>
```

Comment:
The portal item seems to be a zip file, so instead of put it under `Geodatabases` folder, it need to go under `Portal Data` -> `Archives` -> `loudoun_anno.geodatabase` folder.

Due to this issue I cannot build now 😓 waiting for a fix...

Reply:
Fixed!